depthai_helpers/managers.py

```diff
@@ -29,6 +23,21 @@ def convert_disparity_frame(packet, manager):
 def convert_disparity_to_color(disparity, manager):
     return cv2.applyColorMap(disparity, manager.colorMap)
 
+
+def convert_depth_frame(packet, manager):
+    depth_frame = packet.getFrame()
+    dispScaleFactor = getattr(manager, "dispScaleFactor", None)
+    if dispScaleFactor is None:
+        baseline = 75  # mm
```

Comment:
Baseline and focal length should be obtained from EEPROM data, if available, otherwise defaulted to these values.
This is not going to work well on smaller baseline boards.